astropy/coordinates/tests/test_solar_system.py

```diff
@@ -0,0 +1,209 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from ...time import Time
+from ... import units as u
+from ...constants import c
+from ..builtin_frames import GCRS
+from ..earth import EarthLocation
+from ..sky_coordinate import SkyCoord
+from ..solar_system import get_body, get_moon, _apparent_position_in_true_coordinates
+from ...tests.helper import pytest, assert_quantity_allclose, remote_data
+
+try:
+    import jplephem  # pylint: disable=W0611
+except ImportError:
+    HAS_JPLEPHEM = False
+else:
+    HAS_JPLEPHEM = True
+
+try:
+    from skyfield.api import load  # pylint: disable=W0611
+except ImportError:
+    HAS_SKYFIELD = False
+else:
+    HAS_SKYFIELD = True
+
+separation_tolerance_planets = 5*u.arcsec
+separation_tolerance_moon = 5*u.arcsec
+distance_tolerance = 20
```

Comment:
Maybe give the unit of this (as a comment is fine, though even better to use units...)